src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionService.cs

```diff
@@ -17,9 +18,13 @@ namespace Microsoft.CodeAnalysis.Editor.GoToDefinition
     // GoToDefinition
     internal abstract class AbstractGoToDefinitionService : IGoToDefinitionService
     {
-        private readonly IStreamingFindUsagesPresenter _streamingPresenter;
+        /// <summary>
+        /// Presenter used for <see cref="TryGoToDefinition(Document, int, CancellationToken)"/>
+        /// Import lazily as this requires the UI thread.
```

Comment:
Comment can be deleted, since it no longer requires the UI thread. This is now more an optimization just to avoid loading a bunch of code if not necessary.